File: ftp/fsharp/read/ftp_read.fs
// ------------------------------------------------------------------
//	ftp_read.fs
//
//					Mar/14/2013
//
// ------------------------------------------------------------------
open System
open System.Collections.Generic

// ------------------------------------------------------------------
[<EntryPoint>]
let main (args : string[] ) =

  printfn "*** 開始 ***"

  let url_in = "ftp://host_dbase/city/iwate.json"
  let user = "scott"
  let passwd = "tiger"

  let str_json = ftp_manipulate.ftp_to_str_proc (url_in,user,passwd)

  let dict_aa = json_manipulate.json_to_dict_proc (str_json)

  let _ = text_manipulate.dict_display_proc (dict_aa)

  printfn "*** 終了 ***"

  0

// ------------------------------------------------------------------

File: ftp/csharp/read/ftp_read.cs
// ------------------------------------------------------------------
//	ftp_read.cs
//
//					Oct/19/2011
//
// ------------------------------------------------------------------
using	System;
using	System.Collections.Generic;

// ------------------------------------------------------------------
public class ftp_read
{
// ------------------------------------------------------------------
static void Main (string[] args)
{
	Console.WriteLine ("*** 開始 ***");

	string url_in = "ftp://host_dbase/city/iwate.json";
	string user = "scott";
	string passwd = "tiger";

	string str_json = ftp_manipulate.ftp_to_str_proc (url_in,user,passwd);

//	Console.WriteLine (str_json);
	Dictionary <string,Object> dict_aa
		 = json_manipulate.json_to_dict_proc (str_json);

	text_manipulate.dict_display_proc (dict_aa);

	Console.WriteLine ("*** 終了 ***");
}

// ------------------------------------------------------------------
}
// ------------------------------------------------------------------
